.github/workflows/actions.yml

```diff
@@ -75,6 +75,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - uses: julia-actions/setup-julia@v1
+    - name: Ad-hoc fix
+      if: ${{ matrix.os == 'ubuntu-latest' }}
+      run: |
+        # workaround for https://github.com/deepmind/open_spiel/issues/606
+        sudo cp /usr/lib/x86_64-linux-gnu/libstdc++.so.6 $(julia --startup-file=no -E 'using Libdl;print(abspath(joinpath(Libdl.dlpath("libjulia"), "..", "julia")))')
```

Comment:
Can you try copying to `/opt/hostedtoolcache/julia/1.6.1/x64/lib/julia` instead ?

Reply:
The result should be the same. But I'll give it a try.